src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+  const entries = Object.entries(robot);
+
+  for (const entry of entries) {
+    if (fixedRobot.hasOwnProperty(entry[1])) {
+      return null;
+    }
+    fixedRobot[entry[1]] = entry[0];
+  }
```

Comment:
FYI: better to use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

```suggestion
    const [key, value] = entry;

    if (fixedRobot.hasOwnProperty(value)) {
      return null;
    }

    fixedRobot[value] = key;
  }
```